ArtOfIllusion/src/artofillusion/ArtOfIllusion.java

```diff
@@ -272,7 +273,7 @@ public static void newWindow()
   }
 
   /** Create a new window for editing the specified scene. */
-
+  @SuppressWarnings("CallToPrintStackTrace")
```

Comment:
And that is the core of my question. I know that we tabled the discussion about using a logger for the time being. Currently, a lot of AOI code uses printStackTrace() at the time an exception occurs. If we want this to be the "Official" standard way to handle exceptions, we should run through the entire codebase, suppress this warning on any method that contains a `catch` block, and then modify the launchers to redirect stderr to a file.

If we decide to do something different for logging, the warnings are good in that they remind us of where some of our legacy code lives. Either way, we should be consistant.